performance-storage-service/pss_project/api/models/rest/metadata/NoisePageMetadata.py

```diff
@@ -1,3 +1,3 @@
 class NoisePageMetadata(object):
-    def __init__(self,version):
-        self.version = version
\ No newline at end of file
+    def __init__(self, db_version):
+        self.db_version = db_version
```

Comment:
Same comment as above. We probably don't need the `db_`

Reply:
> Same comment as above. We probably don't need the `db_`

Same reply as above.